apps/api/src/server/applications/application/application.service.js

```diff
@@ -222,6 +222,16 @@ export const getCaseRepresentations = async (caseId, pagination, filterAndSort)
 	return representationsRepository.getByCaseId(caseId, pagination, filterAndSort);
 };
 
+/**
+ *
+ * @param {number} caseId
+ * @param {number} repId
+ * @returns {Promise<any>}
```

Comment:
If possible, could we avoid typing responses with `any`?

Reply:
addressed